plugins/image/io/AudioVideo/src/writer/AVWriterPlugin.hpp

```diff
@@ -96,15 +93,28 @@ class AVWriterPlugin : public WriterPlugin
 	void render( const OFX::RenderArguments& args );
 	void endSequenceRender( const OFX::EndSequenceRenderArguments& args );
 
+private:
+	void updateVisibleTools();
+
 	//@{
 	/** Getters which throw exception if format or codec are not found. */
-	std::string getFormatName( const int format ) const;
-	std::string getVideoCodecName( const int codec ) const;
-	std::string getAudioCodecName( const int codec ) const;
+	std::string getFormatName( const size_t formatIndex ) const;
+	std::string getVideoCodecName( const size_t codecIndex ) const;
+	std::string getAudioCodecName( const size_t codecIndex ) const;
 	std::string getPixelFormatName( const std::string& videoCodecName ) const;
 	std::string getSampleFormatName( const std::string& audioCodecName ) const;
 	//@}
 
+	//@{
+	void setFormatParam( const std::string& formatShortName );
+	void setVideoCodecParam( const std::string& videoCodecShortName );
+	void setAudioCodecParam( const std::string& audioCodecShortName );
+	//@}
+
+	// remove some bad values for wrapper/encoder
+	// insert value from preset file which had not taken by an OFX parameter
```

Comment:
Ok, it's documented here, but the message is not clear to me.
Need to use doxygen style.